system_test/aest_sync_SUITE.erl

```diff
@@ -667,14 +667,44 @@ net_split_mining_power(Cfg) ->
 
     ok.
 
-quick_start_stop(Cfg) ->
-    setup_nodes(cluster([n1, n2], #{}), Cfg),
+abrupt_stop_new_node(Cfg) ->
+    RepairTimeout = 30000, % Time allowed for node to repair DB and finish sync
+    Nodes = [n1, n2],
+    setup_nodes(cluster(Nodes, #{}), Cfg),
+    % Start both nodes
+    [start_node(N, Cfg) || N <- Nodes],
+    % Stop node 2 abruptly
+    stop_node(n2, 2000, Cfg),
+    % Restart node 2
     start_node(n2, Cfg),
-    start_node(n1, Cfg),
+    % Check that they synchronize
+    Blocks = wait_for_value({height, 5}, Nodes, RepairTimeout, Cfg),
+    assert_in_sync(Blocks).
+
+abrupt_stop_mining_node(Cfg) ->
+    RepairTimeout = 30000, % Time allowed for node to repair DB and finish sync
+    NodeShutdownTime = proplists:get_value(node_shutdown_time, Cfg),
+    Nodes = [n1, n2],
+    setup_nodes(cluster(Nodes, #{}), Cfg),
+    % Start both nodes
+    [start_node(N, Cfg) || N <- Nodes],
+    % Let them build some chain
+    Blocks = wait_for_value({height, 5}, Nodes, 5 * ?MINING_TIMEOUT, Cfg),
+    assert_in_sync(Blocks),
+    % Stop node 2
+    stop_node(n2, NodeShutdownTime, Cfg),
+    % Start node 2
+    start_node(n2, Cfg),
+    % Stop node 2 abruptly
     stop_node(n2, 2000, Cfg),
-    timer:sleep(2000),
```

Comment:
I think this sleep was needed to really disconnect (otherwise node 2 was not really stopped). But this might have been fixed in the docker framwork